doc/api/stream.md

```diff
@@ -418,14 +418,16 @@ changes:
   `Uint8Array`. For object mode streams, `chunk` may be any JavaScript value
   other than `null`.
 * `encoding` {string} The encoding if `chunk` is a string
-* `callback` {Function} Optional callback for when the stream is finished
+* `callback` {Function} Optional callback for when the stream finishes
+  or errors
 * Returns: {this}
 
 Calling the `writable.end()` method signals that no more data will be written
 to the [`Writable`][]. The optional `chunk` and `encoding` arguments allow one
 final additional chunk of data to be written immediately before closing the
 stream. If provided, the optional `callback` function is attached as a listener
-for the [`'finish'`][] event.
+for the [`'finish'`][] and the `'error'` event. In the latter case, an error
+object will be passed to the `callback` function.
```

Comment:
I think this last sentence could probably be removed. The param description should be enough `callback for when the stream finishes or errors`, this is just implementation details which shouldn't really be in the docs.

Reply:
Removing this. I guess this behaviour is [already defined above](https://github.com/nodejs/node/blame/master/doc/api/stream.md#L278).